### PR TITLE
explain TiDB skip-grant-table and update FAQ

### PR DESCRIPTION
**Pull request by tiancaiamao**
PTAL @shenli @tennix

### CODE REVIEW

File: op-guide/configuration.md [hunk 1]
```diff
@@ -65,9 +65,14 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
...
-+ enable(true) or disable(false) the privilege check.
++ enable(true) or disable(false) the privilege check(for debugging)
++ default: true
++ The value can be (true) or (false). (true) is to enable and (false) is to disable. This option is deprecated and will be removed.
+
+### --skip-grant-table
++ enables anyone to connect without a password and with all privileges
 + default: false
-+ The value can be (true) or (false). (true) is to enable and (false) is to disable. This feature is a work in progress, so it's disabled by default. It will be enabled by default when finished.
++ The value can be (true) or (false). This option is usually used to reset password, enable it need the root privileges on the OS.
```
Comment (tennix) [inline op-guide/configuration.md]: s/enable/enabling/
s/need/needs/

**Comment by tiancaiamao** [comment]
PTAL @tennix  @queenypingcap

File: FAQ.md [hunk 2]
```diff
@@ -66,12 +66,6 @@ TiDB follows MySQL user authentication mechanism. You can create user accounts a
   GRANT SELECT ON \*.\* TO  'test'@'localhost';
   ```
 
-Note: There are following differences between TiDB and MySQL in user account creating and authorizing:
-
-+ You can use the "user@hostname" grammar to create accounts. In this grammar, “hostname” supports exact match like “192.168.199.1” and full match like “%”, but doesn’t support prefix match like “192.168.%”.
-
-+ To be compatible with the existing MySQL businesses, TiDB supports user authorization but it only supports the authorization grammar and recording the authorization in the system table. TiDB checks the authorization for the DropTable statement only. It does not check the authorization for other statements.
```
Comment (queenypingcap) [inline FAQ.md]: recording -> records

Comment (tiancaiamao) [inline FAQ.md]: This paragraph is deleted.

**Comment by queenypingcap** [comment]
LGTM

**Comment by tennix** [comment]
LGTM

**Pull request merged**